stdlib/camlinternalFormat.ml

```diff
@@ -537,6 +544,16 @@ let bprint_fmt buf fmt =
     | Theta rest ->
       buffer_add_char buf '%'; bprint_ignored_flag buf ign_flag;
       buffer_add_char buf 't'; fmtiter rest false;
+    | Custom (arity, _, rest) ->
+      let n = int_of_custom_arity arity in
+      if n > 0 then begin
+        buffer_add_char buf '%'; bprint_ignored_flag buf ign_flag;
+        buffer_add_char buf '?';
+        for _i = 1 to n - 1 do
+          buffer_add_string buf "%?"
```

Comment:
Yes, the flag would only appear if you (1) added an "ignored" counterpart of Custom_arity and (2) generated code using it. I think it's reasonable to assume that you may want to do that in the future, eg. to support `%_{sexp:Time.t}`.